src/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]>      <html class="no-js"> <!--<![endif]-->
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>LeaderBoard Project</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="" />
+  </head>
+  <body>
+    <!--[if lt IE 7]>
+      <p class="browsehappy">
+        You are using an <strong>outdated</strong> browser. Please
+        <a href="#">upgrade your browser</a> to improve your experience.
+      </p>
+    <![endif]-->
+
+    <main class="main">
+      <h1>Leaderboard</h1>
+
+      <section class="main-section">
+       
+				<section class="display-section">
+          <div class="display-title">
+            <h2>Recent Scores</h2>
+            <button>Refresh</button>
+          </div>
+
+          <table>
+            <tr>
+              <td>Namely McNamerson</td>
+              <td>100</td>
+            </tr>
+						
+						<tr>
+              <td>Namely McNamerson</td>
+              <td>100</td>
+            </tr>
+
+						<tr>
+              <td>Namely McNamerson</td>
+              <td>100</td>
+            </tr>
+
+						<tr>
+              <td>Namely McNamerson</td>
+              <td>100</td>
+            </tr>
+
+						<tr>
+              <td>Namely McNamerson</td>
+              <td>100</td>
+            </tr>
+
+						<tr>
+              <td>Namely McNamerson</td>
+              <td>100</td>
+            </tr>
+         
+					</table>
+        </section>
+
+				<section class="add-section">
+					<div>
+						<h2>Add your Score</h2>
+					</div>
+					<form action="" class="add-form">
+						<input type="text" placeholder="Your name"/>
+						<input type="text" placeholder="Your score"/>
+						<button>Submit</button>
+					</form>
+				</section>
+      
+			</section>
+    </main>
+
+    <script src="" async defer></script>
+  </body>
+</html>
```

Comment:
While you did a good job implementing this it would be nice if you stick to the wire frame design [wireframe ](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png) kindly style your wire frame exactly as it looks in the original wireframe. You can also add a border around your score board.

This is your design: 


![wireframe 2](https://user-images.githubusercontent.com/79506105/203992185-15ca2f77-2c4b-416f-a18d-9b84ec118dad.PNG)

Please stick to this design: 

![original wireframe](https://user-images.githubusercontent.com/79506105/203992323-fd7f83e3-09e1-4c45-b1bd-6e118f1c0331.PNG)

NOTE: Please pay attention to detail.

Reply:
Fixed.